docker/default-config/application.properties

```diff
@@ -1,36 +1,44 @@
 server.contextPath=/
 server.port=8080
 server.use-forward-headers=true
-server.forward-headers-strategy=framework
+
+springdoc.swagger-ui.enabled=true
+springdoc.swagger-ui.path=index.html
+springdoc.swagger-ui.tagsSorter=alpha
 
 springdoc.api-docs.path=/api-docs
 springdoc.api-docs.enabled=true
+
 springdoc.swagger-ui.tagsSorter=alpha
-springdoc.packagesToScan=gov.nasa.pds.api.registry.controller
+springdoc.packagesToScan=gov.nasa.pds.api.registry.controllers
 springdoc.pathsToMatch=/**
-
+server.forward-headers-strategy=framework
+management.endpoints.web.exposure.include=*
 spring.jackson.serialization.WRITE_DATES_AS_TIMESTAMPS=false
 
 debug=false
 logging.level.root = INFO
+log4j.logger.org.springframework=DEBUG
```

Comment:
Do we want to keep this at `DEBUG`? It could get a little "noisy"

Reply:
Right, I don't remember why we had that. I removed it.